tests/foreman/cli/test_repository.py

```diff
@@ -2417,20 +2444,32 @@ def test_positive_local_directory_sync(self):
 
         :expectedresults: entire directory is synced
 
-        :CaseAutomation: notautomated
         """
+        # Making Setup For Creating Local Directory using Pulp Manifest
+        ssh.command("yum -y install python-pulp-manifest")
+        ssh.command("mkdir {}".format(CUSTOM_LOCAL_FOLDER))
+        ssh.command("touch {0} && pulp-manifest {1}".
+                    format(CUSTOM_LOCAL_FILE, CUSTOM_LOCAL_FOLDER))
+        repo = make_repository({
+            'content-type': 'file',
+            'product-id': self.product['id'],
+            'url': Path(CUSTOM_LOCAL_FOLDER).as_uri(),
```

Comment:
Local and remote machine can be different OSes (we know that remote is always Unix like file system). Are you sure this should work even if the script is run from a Windows machine?

Reply:
@ldjebran : Do you have any example where I can look for this ?

Reply:
@omkarkhatavkar  https://docs.python.org/3/library/pathlib.html
Please pay attention to this sentence: 
"""
If you’ve never used this module before or just aren’t sure which class is right for your task, Path is most likely what you need. It instantiates a concrete path **for the platform the code is running on**.
"""

Reply:
@omkarkhatavkar in our case we can run in other platform than the path is for.

Reply:
@omkarkhatavkar to avoid any confusion we can simply write:
```
...
'url':  'file://{0}'.format(CUSTOM_LOCAL_FOLDER)
...
``` 
What do you think?

Reply:
Ok, let me see how I can fix this.